cdc/log.cc

```diff
@@ -228,8 +228,23 @@ namespace cdc {
 using operation_native_type = std::underlying_type_t<operation>;
 using column_op_native_type = std::underlying_type_t<column_op>;
 
+static sstring cdc_log_suffix = "_scylla_cdc_log";
+
+bool is_log_name(const std::string_view& table_name) {
```

Comment:
is it used anywhere outside of log.cc? Perhaps it can be made static and removed from log.hh

Reply:
It is only used in log.cc. Converted it into a file-local static function.